src/goImplementations.ts

```diff
@@ -70,8 +70,8 @@ export class GoImplementationProvider implements vscode.ImplementationProvider {
 
 					let guruOutput = <GuruImplementsOutput>JSON.parse(stdout.toString());
 					let results: vscode.Location[] = [];
-					let addResults = list => {
-						list.forEach(ref => {
+					let addResults = (list: any) => {
+						list.forEach((ref: any) => {
```

Comment:
>ref [](start = 20, length = 3)

this seems to be `to_method`, `to`, `from` etc from below which are coming from a `GuruImplementsOutput` type, there is no type for those from there ?